skrub/tests/test_fuzzy_join.py

```diff
@@ -9,19 +9,33 @@
 from sklearn.feature_extraction.text import HashingVectorizer
 
 from skrub import fuzzy_join
+from skrub._dataframe._polars import POLARS_SETUP
+from skrub._dataframe._test_utils import is_namespace_polars
 
+MODULES = [pd]
+ASSERT_TUPLES = [(pd, assert_frame_equal)]
 
+if POLARS_SETUP:
+    import polars as pl
+    from polars.testing import assert_frame_equal as assert_frame_equal_pl
+
+    MODULES.append(pl)
+    ASSERT_TUPLES.append((pl, assert_frame_equal_pl))
```

Comment:
let's leave it for another pr but I think we should reorganize this a bit to put this kind of setup either in the `_test_utils.py` or in `conftest.py` and avoid duplicating it for each test module